web/.env.cypress

```diff
@@ -1,9 +1,9 @@
 BROWSER=none
 PORT=3030
-REACT_APP_API_BASE_URL=http://localhost:8080/api
-REACT_APP_HIDE_DISCLAIMER=false
-REACT_APP_KEYCLOAK_AUTH_URL=https://keycloak-auth-url.ca
-REACT_APP_SM_LOGOUT_URL=https://sm-logout-url.ca
-REACT_APP_KEYCLOAK_REALM=8wl6x4cp
-REACT_APP_KEYCLOAK_CLIENT=wps-web
-REACT_APP_PMTILES_BUCKET=https://nrs.objectstore.gov.bc.ca/gpdqha/psu/pmtiles/
```

Comment:
i hadn't noticed this. so cypress is using the real bucket but not other tests

Reply:
Other tests could use the real bucket (these are public, I forgot)

Reply:
done